src/test/java/com/jadajabajava/services/VacancyServiceTest.java

```diff
@@ -0,0 +1,83 @@
+package com.jadajabajava.services;
+
+import com.jadajabajava.entities.Vacancy;
+import com.jadajabajava.exceptions.CommonServiceException;
+import com.jadajabajava.exceptions.VacancyProviderException;
+import com.jadajabajava.providers.VacancyProvider;
+import com.jadajabajava.repositories.VacancyRepository;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.context.MessageSource;
+
+import java.util.List;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.fail;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class VacancyServiceTest {
+
+    @Mock
+    private VacancyRepository vacancyRepository;
+
+    @Mock
+    private VacancyProvider vacancyProvider;
+
+    @Mock
+    private MessageSource messageSource;
+
+    @InjectMocks
+    private VacancyService vacancyService;
+
+    @Mock
+    private Vacancy mockedVacancyOne;
+
+    @Mock
+    private Vacancy mockedVacancyTwo;
+
+    @Test
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/jada-jaba-java/job-seeker-service/pullRequest?prid=5720132)